### ISSUE TITLE
How can I merge outputs of two layers as an input of another? 

### ISSUE THREAD

**Issue by mjqgit** [fchollet/keras#7048]
is the code below correct?
x = Dense(128, activation='relu')(x)
preds = Dense(len(labels_index), activation='softmax')(x)
x = concatenate([x, preds])
x = Dense(128, activation='relu')(x)
x = Dense(128, activation='relu')(x)
x = Dense(128, activation='relu')(x)
sub_preds = Dense(len(sub_labels_index), activation='softmax')(x)

**Comment by fchollet** [comment]
You should use a `merge` function, such as `concatenate` in your example (or `sum`, etc). For code correctness checks, please ask on StackOverflow.
